### PR TITLE
Close about dialog to see the actual interactive

### PR DESCRIPTION
**Pull request by eireland**
Added code to default.rb to close the About dialog and take another screenshot so interactive isn't hidden behind the dialog.
Had to remove the code for closing dialogs in the target interactives

### CODE REVIEW

File: interactive-tests/default.rb [hunk 1]
```diff
@@ -13,3 +23,11 @@
...
 sleep 1.5
 $test.save_screenshot
+
+close_dialog_size = $test.driver.find_elements(:css=>'.about-dialog > .ui-dialog-titlebar > .ui-dialog-titlebar-close').size #checks to see if there is an about dialog
+if close_dialog_size > 0 && visible #some interactives have hidden about dialogs so have to check if they are visible
+    puts "about box is visible"
+  $test.driver.find_element(:css=>'.about-dialog > .ui-dialog-titlebar > .ui-dialog-titlebar-close').click
+  sleep 1.5
+  $test.save_screenshot
+end
```
Comment (pjanik) [inline interactive-tests/default.rb]: Note that there's already a function that closes About dialog:
https://github.com/concord-consortium/lab-selenium-tests/blob/master/lib/test_api.rb#L38-L46

I guess it could be used (you removed calls to it below). Or if you need to check if the dialog is present first, I guess we could add a new one there, something like `close_about_dialog_if_present`.

File: interactive-tests/interactives_interactions_targetFreePlay.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # Run basic test first.
 load 'interactive-tests/default.rb', true
-$test.click_element('#main-help-icon')
-$test.save_screenshot
```
Comment (pjanik) [inline interactive-tests/interactives_interactions_targetFreePlay.rb]: Why was this removed? It doesn't seem to be related to About dialog. It checks help system actually.

File: interactive-tests/interactives_interactions_targetGameDist.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Run basic test first.
 load 'interactive-tests/default.rb', true
-$test.click_element('#main-help-icon')
-$test.save_screenshot
+
```
Comment (pjanik) [inline interactive-tests/interactives_interactions_targetGameDist.rb]: Why was this removed? It doesn't seem to be related to About dialog. It checks help system actually.

File: interactive-tests/default.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+def visible
+  style = $test.driver.find_element(:css=>'.about-dialog').attribute("style")
+  if style.include?('block')
+    return true
+  else
+    return false
+  end
+
+end
+
```
Comment (pjanik) [inline interactive-tests/default.rb]: This function was added between comments and the code that comments were referring to. ;) But I guess it'd be nice to add this logic to test_api.rb as there's already similar function.

**Review by pjanik — CHANGES_REQUESTED** [review]
It looks good, but not everything was clear to me. If you're struggling with some changes, let me know and I guess I'll be able to quickly do that using your work. Thanks!

**Comment by eireland** [comment]
Moved the logic to check if about dialog is present and visible to test_api.
Added the click to overlay back to target interactives.

**Comment by pjanik** [comment]
Looks great now, thanks!
